templates/about/index.html

```diff
@@ -7,115 +7,140 @@
+          </div>
+        </li>
+        <li class="p-list__item row u-no-padding--bottom">
+          <div class="col-2">
+            <h4 class="p-heading--5">Snapcraft</h4>
+          </div>
+          <div class="col-4">
+            <p class="u-no-margin--bottom">Snapcraft is the command and the framework used to build and publish snaps.</p>
+          </div>
+        </li>
+      </ul>
     </div>
   </div>
-  <div class="p-strip is-shallow">
-    {{
-      image(
-        url="https://assets.ubuntu.com/v1/c6335a68-01+A+universal+store+x2.png",
-        alt="",
-        width="725",
-        height="240",
-        hi_def=True
-      ) | safe
-    }}
-    <h2>
-      A universal store
-    </h2>
-    <h3 class="p-heading--4">
-      Snaps run not only on desktops but also on IoT devices, servers and clouds
-    </h3>
-    <p>
-      Snaps work across Linux on many distributions and versions. Bundle your dependencies and assets, to simplify installs to a single standard command.
-    </p>
-    <a href="/iot" class="p-button">Learn more about snaps in IoT</a>
-  </div>
-  <div class="p-strip is-shallow">
-    {{
-      image(
-        url="https://assets.ubuntu.com/v1/1dff8c80-02+Built%2C+published+and+hosted+for+free+x2.png",
-        alt="",
-        width="725",
-        height="240",
-        hi_def=True
-      ) | safe
-    }}
-    <h2>Built, published and hosted for free</h2>
-    <h3 class="p-heading--4">
-      Our build infrastructure and the Snap Store makes building and publishing snaps a breeze
-    </h3>
-    <p>
-      Whether your project is hosted on GitHub or not, you can leverage our build system to both build and release to the edge channel, ensuring users always stay up&dash;to&dash;date.
-    </p>
-    <a href="/build" class="p-button">Learn more about our Build service</a>
-  </div>
-  <div class="p-strip is-shallow">
-    {{
-      image(
-        url="https://assets.ubuntu.com/v1/9aeb6b02-03+Software+distribution+in+your+hands+x2.png",
-        alt="",
-        width="725",
-        height="240",
-        hi_def=True
-      ) | safe
-    }}
-    <h2>Software distribution in your hands</h2>
-    <h3 class="p-heading--4">
-      With Snapcraft, you are in control of when and how you publish your Snap
-    </h3>
-    <p>
-      With channels users can find their preferred balance between stability and the latest features. Though nothing is enforced, users expect the “Stable” channel to be well tested and updated less frequently, whereas the “Edge” channel is where they&rsquo;ll get the latest release as soon as it&rsquo;s available.
-    </p>
-    <a href="/docs/channels" class="p-button">Learn more about channels</a>
+</section>
+
+<section class="p-strip">
+  <div class="u-fixed-width">
+    <h2>Official snaps from publishers</h2>
+    <div class="p-logo-section">
+      <div class="p-logo-section__items u-no-margin--bottom">
```

Comment:
We might need those for images? What do you think?
<div class="row">
<div class="col-small-2 col-medium-3 col-2 col-logo u-align--center">